x-pack/legacy/plugins/siem/public/components/events_viewer/events_viewer.tsx

```diff
@@ -171,7 +173,7 @@ const EventsViewerComponent: React.FC<Props> = ({
                         {headerFilterGroup}
                       </HeaderSection>
 
-                      {utilityBar?.(totalCountMinusDeleted)}
+                      {utilityBar?.(refetch as inputsModel.Refetch, totalCountMinusDeleted)}
```

Comment:
Looks like this `as` cast isn't necessary -- I was able to remove it without error.

```suggestion
                      {utilityBar?.(refetch, totalCountMinusDeleted)}
```